src/pages/Profile/index.js

```diff
@@ -23,90 +32,361 @@ import avatar from '../../assets/avatar.svg';
+                          <GridItem span={4}>
+                            <h1>
+                              <b>Profile Picture</b>
+                            </h1>
+                            <div className={styles.avatarDiv}>
+                              <img src={avatar} alt="avatar" className={styles.avatar} />
+                            </div>
+                          </GridItem>
+                          <GridItem span={4}>
+                            <h1>
+                              <b>Display Name</b>
+                            </h1>
+                            {editView === false ? (
+                              <TextContent>
+                                <Text component={TextVariants.h5}>{displayName}</Text>
+                              </TextContent>
+                            ) : (
+                              <TextInput
+                                ref={name}
+                                placeholder={displayName}
+                                type="text"
+                                aria-label="text input example"
+                                className={styles.editInput}
+                              />
+                            )}
+                          </GridItem>
+                          <GridItem span={4}>
+                            <h1>
+                              <b>Position / Role</b>
+                            </h1>
+                            {editView === false ? (
+                              <TextContent>
+                                <Text component={TextVariants.h5}>{positionVal}</Text>
+                              </TextContent>
+                            ) : (
+                              <TextInput
+                                ref={position}
+                                placeholder={positionVal}
+                                type="text"
+                                aria-label="text input example"
+                                className={styles.editInput}
+                              />
+                            )}
+                          </GridItem>
+                        </Grid>
+                        <Grid span={12} className={styles.grid}>
+                          <GridItem span={4}>
+                            <h1>
+                              <b>ID</b>
+                            </h1>
+                            {editView === false ? (
+                              <TextContent>
+                                <Text component={TextVariants.h5}>RH329821</Text>
+                              </TextContent>
+                            ) : (
+                              <TextInput
+                                value="RH329821"
+                                type="text"
+                                isReadOnly
+                                aria-label="RH329821"
+                                className={styles.editInput}
+                              />
```

Comment:
Why is ID not in the state?

Reply:
I thought the ID of an employee won't change. Should I make it editable?

Reply:
+1,  if ID is a read-only property we should consider referencing it as a prop.

Reply:
@gurbirkalsi @dbutenhof added the ID in the mock file as a prop. Removed the reference from the state.

Reply:
I don't know whether "ID" is or should be static; is the meaning strongly defined in the mocks anywhere? It's not even necessarily true that an employee ID doesn't change over time; for example when Compaq absorbed Digital, the older low numbered Digital badge numbers (including mine) overlapped and got changed. It would have been mildly annoying to be using a database system like this that didn't allow correcting that.